README.md

```diff
@@ -85,19 +85,20 @@ By default models are exported with ensemble=False and Fast=True
   - 62 = rife-v4.20 (ensemble=True)
   - 63 = rife-v4.21 (ensemble=False)
   - 64 = rife-v4.22 (ensemble=False)
-  - 65 = rife-v4.22-lite(ensemble=False)
-  - 66 = rife-v4.24 (ensemble=False)
-  - 67 = rife-v4.24 (ensemble=True)
-  - 68 = rife-v4.25 (ensemble=False)
-  - 69 = rife-v4.25-lite (ensemble=False)
-  - 70 = rife-v4.25-heavy (ensemble=False)
-  - 71 = rife-v4.26 (ensemble=False)
+  - 65 = rife-v4.22-lite (ensemble=False)
+  - 66 = rife-v4.23-beta (ensemble=False)
+  - 67 = rife-v4.24 (ensemble=False)
+  - 68 = rife-v4.24 (ensemble=True)
+  - 69 = rife-v4.25 (ensemble=False)
+  - 70 = rife-v4.25-lite (ensemble=False)
+  - 71 = rife-v4.25-heavy (ensemble=False)
+  - 72 = rife-v4.26 (ensemble=False)
 
   ## My experimental custom models (only works with 2x)
 
-  - 72 = sudo_rife4 (ensemble=False / fast=True)
-  - 73 = sudo_rife4 (ensemble=True / fast=False)
-  - 74 = sudo_rife4 (ensemble=True / fast=True)
+  - 73 = sudo_rife4 (ensemble=False / fast=True)
```

Comment:
Everytime we add new models, the compatibility is broken for these models.
I suggestion is to use 5000+ numbers as ID for these sudo models.